Source/WebCore/page/Quirks.cpp

```diff
@@ -1287,7 +1287,7 @@ Quirks::StorageAccessResult Quirks::triggerOptionalStorageAccessQuirk(Element& e
 
     static NeverDestroyed<UserScript> kinjaLoginUserScript { "function triggerLoginForm() { let elements = document.getElementsByClassName('js_header-userbutton'); if (elements && elements[0]) { elements[0].click(); clearInterval(interval); } } let interval = setInterval(triggerLoginForm, 200);"_s, URL(aboutBlankURL()), Vector<String>(), Vector<String>(), UserScriptInjectionTime::DocumentEnd, UserContentInjectedFrames::InjectInTopFrameOnly, WaitForNotificationBeforeInjecting::Yes };
 
-    if (eventType == eventNames().clickEvent) {
+    if (eventType == eventNames().clickEvent || eventType == eventNames().auxclickEvent) {
```

Comment:
Also use `MouseEvent::isClickOrAuxClickEvent()` here?

Reply:
I can't, since we only have access to the type and not the event

Reply:
Oh, I see. Maybe? Or not, up to you.

```cpp
bool MouseEvent::isClickOrAuxClickEvent(Event& e) { return MouseEvent::isClickOrAuxClickEventType(e.type()); }

bool MouseEvent::isClickOrAuxClickEventType(const AtomString& type)
{
    return type == eventNames().clickEvent || type == eventNames().auxclickEvent;
}
```

Reply:
I think the overloading is a clever way to do both.

Reply:
ah, I didn't realize the suggestion was to use both methods!